include/mbedtls/ssl_internal.h

```diff
@@ -158,17 +173,43 @@
 #error Bad configuration - record content too large.
 #endif
 
-#if MBEDTLS_SSL_PAYLOAD_LEN > 16384 + 2048
-#error Bad configuration - protected record payload too large.
+#if MBEDTLS_SSL_IN_CONTENT_LEN > MBEDTLS_SSL_MAX_CONTENT_LEN
+#error Bad configuration - incoming record content should not be larger than MBEDTLS_SSL_MAX_CONTENT_LEN.
```

Comment:
I know this is replacing like for like, but all other `#error` directives encapsulate their error messages in strings. Offhand I'm not sure if there's any technical difference.

For consistency, it would be nice for it to be the same.

Reply:
What follows `#error` must be lexically valid, so you can't portably have something like `#error You can't do this`. But an invalid `#error` directive is likely to terminate the compilation with a message containing what follows the `#error` directive, so it doesn't matter much.